### ISSUE TITLE
Is it possible to supply the auth token as a parameter?

### ISSUE THREAD

**Issue by tbell511** [pusher/pusher-js#710]
Quick question...

Instead of supplying an auth URL that returns the token, is there is way to set the token to avoid having that additional request?

I'm asking because I can easily supply the auth token in my apps init call. That would prevent me from having to add an additional endpoint to my api.

Thanks.

**Comment by tbell511** [comment]
Seems like I may have found the answer.

> customHandler (Function) - When present, this function is called instead of a request being made to the endpoint specified by userAuthentication.endpoint.

**Comment by benw-pusher** [comment]
Yep using the customHander function would be the correct way to achieve this.
